25.0/Dockerfile

```diff
@@ -4,6 +4,7 @@
 #   NOTE: Unlike our other images this one is NOT prefixed with `v`,
 #           as many things (like download URLs) use this form instead.
 ARG VERSION=25.0.0
```

Comment:
What is the point of keeping `ARG` as 25.0.0, and adding a new `URLVERSION` argument? `25.0.0` is incorrect.

Reply:
thats true.. Trying to figure out why the whole thing wasn't building